### PR TITLE
ci: deploy to netlify

### PR DESCRIPTION
**Pull request by barmac**
Related to https://github.com/bpmn-io/internal-docs/issues/839

### CODE REVIEW

**Comment by barmac** [comment]
Once this is merged, we should configure the domains with the netlify account.

File: .github/workflows/PRODUCTION.yml [hunk 1]
```diff
@@ -24,11 +24,11 @@ jobs:
         GTM_KEY: ${{ secrets.GTM_KEY }}
       run: npm run all
     - name: Deploy
+      uses: nwtgck/actions-netlify@v2.0
```
[the remark this answers is not in the document]
Comment (barmac) [reply .github/workflows/PRODUCTION.yml:27]: Good point! I don't know why I thought it's the official one 🙄

File: .github/workflows/STAGING.yml [hunk 2]
```diff
@@ -24,11 +24,9 @@ jobs:
...
+        args: deploy --dir=./dist --prod
       env:
-        HOST: ${{ secrets.BPMN_IO_HOST }}
-        USER: ${{ secrets.BPMN_IO_USER }}
-        SSHPASS: ${{ secrets.BPMN_IO_PASSWORD }}
-        SITE_PATH: ${{ secrets.BPMN_IO_SITE_PATH }}
-      run: |
-        sshpass -e ssh -oStrictHostKeyChecking=no "$USER@$HOST" "rm -rf $SITE_PATH/*"
-        sshpass -e scp -r dist/* "$USER@$HOST:$SITE_PATH"
+        NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
```
Comment (nikku) [inline .github/workflows/STAGING.yml:31]: Should be updated with different site ID? Or do you plan to set this per workflow / action?

Comment (barmac) [reply .github/workflows/STAGING.yml:31]: It's set per environment.

Comment (nikku) [reply .github/workflows/STAGING.yml:31]: :+1:

**Review by nikku — APPROVED** [review]

**Comment by nikku** [comment]
What we could do to aid the transition is to keep the existing deployment. Unless, of course, we're quick to re-direct the DNS to netlify.

**Comment by barmac** [comment]
I could handle the DNS part when this PR is on `main` but not `live` yet.

**Comment by nikku** [comment]
It is on main now :rocket: :slightly_smiling_face:

**Pull request merged**